docs/source/glossary.rst

```diff
@@ -14,6 +14,13 @@ Glossary
         Agnostic of frontend implementation, it recieves requests from the frontend and forwards them to the corresponding Service.
         In addition it is also responsible for catching errors, and returning a human readable error message to the frontend.
 
+    Background Run ID
```

Comment:
```suggestion
    Background Run Number
```
I think we are sticking with Run Number instead of Run Id, although there is a definition of RunId its mostly a catch all for the instance we missed.